tests/benchmark/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM google/python
+
+RUN hg clone https://hg.python.org/benchmarks /app/benchmarks
```

Comment:
Yep, this is just a start. I want to have both available, but this one is slightly more straightforward to setup so I did it first.